lib/App/Seacan.pm

```diff
@@ -100,7 +101,7 @@ sub install_cpan {
     my $cpanm_command = path( $self->seacan_perlbrew_root, "bin", "cpanm")->stringify;
     my $perl_command = $self->seacan_perl;
 
-    $, = " ";
+    local $OUTPUT_FIELD_SEPARATOR = q{ };
```

Comment:
I don't mind. I just wasn't even sure if I remember what that variable was, so I just used English for it.